docs/cve-creation-guide.md

```diff
@@ -0,0 +1,189 @@
+
+### Affected component(s)
+
+This section looks for details on what exactly caused the issue.
+In the case of older issues that have been fixed it is can be fairly easy to extract the affected filenames and other relevant information.
+For _CPANSA-Mojolicious-2018-01_ 
+
+1. lib/Mojo/Cookie/Response.pm
+1. lib/Mojo/UserAgent/CookieJar.pm
+
+The field requires comma separated values:
+
+1. Affected component(s): **_lib/Mojo/Cookie/Response.pm, lib/Mojo/UserAgent/CookieJar.pm_**
+
+### Attack vector(s)
+
+The attack vector can be difficult to determine or articulate consisely.
+For _CPANSA-Mojolicious-2018-01_ it is somewhat unclear whether the cookie for site A is leaked to Site B or simply that old cookies are sent to the web server to which you are connecting.
+
+However, it would seem that connecting to a web server may send old cookie data to that server.
+
+1. Attack vector(s): **_Connecting to a web server using a vulnerable version submits information (cookies) that should not have been submitted_**
+
+### Suggested description of the vulnerability for use in the CVE
+
+If you are using CPAN::Audit as the source information you can likely use the information listed there.
+It may also be a good idea to reference the Issue or Pull Request if either of them provide additional relevant information.
+
+
+1. Suggested description of the vulnerability for use in the CVE:
+
+   Consider that this should be useful as an email subject, and searchable.
+   The following example roughly matches the de-factor standard formatting for Perl module vulnerabilities.
+
+      **_Mojolicious before 7.66 for Perl has Cross Domain Cookie Leakage in Mojo::UserAgent::CookieJar_**
+
+   - "Mojolicious" is the name of the package and should be first in the description
+   - "before 7.66" implies that it has been fixed in later versions
+   - "for Perl" identifies this as a perl package, allowing discoverability
+   - "Cross Domain Cookie Leakage" is a short description of the vulnerability
+   - "Mojo::UserAgent::CookieJar" is the vulnerable component/function inside the package (optional)
+
+
+### Discoverer(s)/Credits
+
+On occasion, the discoverer and or credit may be disputed.
+It is likely good practice to error on the side of caution and report the facts with references.
+Sometimes a maintainer will specifically credit a reporter or discoverer but mentioning the issue as logged by ... might be best.
+In the case of _CPANSA-Mojolicious-2018-01_ the issue listed in https://github.com/mojolicious/mojo/issues/1185 was logged by @expbbc.
+
+Interestingly @expbbc did not mention security in the initial issue.
+They did however mention that it might be considered a security issue in the [Pull Request](https://github.com/mojolicious/mojo/pull/1192)
+
+1. Discoverer(s)/Credits: **_Paul Robins (@expbbc)_**
+
+### Reference(s)
+
+If you are using CPAN::Audit as the source information it provides reference URLs that can be used here.
```

Comment:
Any public URL i.e. mailing list link, github issue, PR, will be a relevant reference.

Reply:
Fixed, references any public URL